AirCasting/Settings/SettingsView.swift

```diff
@@ -106,6 +106,9 @@ struct SettingsView: View {
             Section() {
                 navigateToAppConfigurationButton
                 shareLogsButton
+                Text("Crashlytics integration testing:")
+                crashButton
```

Comment:
This is added to test logging?

Reply:
yes, we can force-crash the app here and see if crashlytics have the logs. Note that it can take a while for crashlytics to register a crash